metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -367,6 +435,36 @@ private void checkSnapshotContents(
         );
     }
 
+    private void checkSnapshotSubcontent(
```

Comment:
Can you add a short JavaDoc for this?

One question here is whether this function considers it an error if the iterator contains more records than are in "expected" I think the answer is no, but can you document this?

Reply:
Added a comment to the method.